scripts/qiita_test_install

```diff
@@ -0,0 +1,396 @@
+                                          string_acceptable_version))
+
+    def test_redis_version(self):
+        if not self.redis_version:
+            self.assertTrue(False, 'redis not running or configured')
+
+        acceptable_version = (2, 8, 17)
+        string_acceptable_version = '.'.join(map(str, acceptable_version))
+        version = tuple(map(int, self.redis_version.split('.')))
+
+        self.assertTrue(acceptable_version <= version,
+                        'Unsupported redis version. You have %s but %s is '
+                        'requiered' % ('.'.join(map(str, version)),
+                                       string_acceptable_version))
+
+system_info_header = """
+System information
+==================
+"""
+
+dependency_info_header = """
+Dependency versions
+===================
+"""
+
+qiita_config_header = """
+Qiita config
+============
+For definitions of these settings and to learn how to configure Qiita, visit:
+ https://github.com/biocore/qiita/blob/master/INSTALL.md#install
+"""
+
+qiita_config_tests_header = """
+Qiita version and configuration tests
+=====================================
+"""
+
+
+def main():
+    system_info = [
+        ("Platform", platform),
+        ("Python version", python_version.replace('\n', ' ')),
+        ("Python executable", executable)]
+    max_len = max([len(e[0]) for e in system_info])
+    print system_info_header
+    for v in system_info:
+        print "%*s:\t%s" % (max_len, v[0], v[1])
+
+    version_info = [
+        ('click library version', click_lib_version),
+        ('future library version', future_lib_version),
+        ('numpy library version', numpy_lib_version),
+        ('pandas library version', pandas_lib_version),
+        ('tornado library version', tornado_lib_version),
+        ('redis library version', redis_lib_version),
+        ('bcrypt library version', bcrypt_lib_version),
+        ('pyparsing library version', pyparsing_lib_version),
+        ('networkX library version', networkx_lib_version),
+        ('WTForms library version', wtforms_lib_version),
+        ('qiime library version', qiime_lib_version + ', for more info use '
+         'print_qiime_config.py'),
+        ('mock library version', mock_lib_version),
+        ('MOI library version', moi_lib_version),
+        ('MOI config file path', environ['MOI_CONFIG_FP']),
+        ('IPython library version', ipython_lib_version),
+        ('psycopg2 library version', psycopg2_lib_version),
+        ('Qiita core library version', qiita_core_lib_version),
+        ('Qiita db library version', qiita_db_lib_version),
+        ('Qiita pet library version', qiita_pet_lib_version),
+        ('Qiita ware library version', qiita_ware_lib_version)
+    ]
+    max_len = max([len(e[0]) for e in version_info])
+    print dependency_info_header
+    for v in version_info:
+        print "%*s:\t%s" % (max_len, v[0], v[1])
+
+    qiita_config = ConfigurationManager()
+    try:
```

Comment:
This needs an exists test after the try-except to make sure the file actually exists at the path given